domehunter/enumerations.py

```diff
@@ -0,0 +1,101 @@
+from enum import Flag
+
+
+class Direction(Flag):
+    CW = True
+    CCW = False
+    none = None
+
+
+class LED_light(Flag):
+    POWER = 0b100000000000000000
+    COMMS = 0b010000000000000000
+    WARN = 0b001000000000000000
+    INPUT_1 = 0b000100000000000000
+    INPUT_2 = 0b000010000000000000
+    INPUT_3 = 0b000001000000000000
+    RELAY_3_NC = 0b000000100000000000
+    RELAY_3_NO = 0b000000010000000000
+    RELAY_2_NC = 0b000000001000000000
+    RELAY_2_NO = 0b000000000100000000
+    RELAY_1_NC = 0b000000000010000000
+    RELAY_1_NO = 0b000000000001000000
+    OUTPUT_3 = 0b000000000000100000
+    OUTPUT_2 = 0b000000000000010000
+    OUTPUT_1 = 0b000000000000001000
+    ADC_3 = 0b000000000000000100
+    ADC_2 = 0b000000000000000010
+    ADC_1 = 0b000000000000000001
+
+
+class ReturnCode(Flag):
```

Comment:
Unless you anticipate doing bitwise operations with these return codes then plain `Enum` class is more appropriate than `Flag`. Alternatively if you anticipate using the integer values a lot then `IntEnum` is probably best.